src/js/pyodide.ts

```diff
@@ -280,6 +280,8 @@ export async function loadPyodide(
   Module.locateFile = (path: string) => {
     throw new Error("Didn't expect to load any more file_packager files!");
   };
+  Module["__emscripten_get_now"] = Module["_emscripten_get_now"];
```

Comment:
Could you add a comment about these lines? I imagine something from C calls this JS function? What's going on with the `sig` though?

Reply:
"v" means that the function takes 0 arguments and returns void (first letter is return type, subsequent letters are argument types). The signature is consumed here:

https://github.com/emscripten-core/emscripten/blob/7e538a419c3649f3a540a57beab347aa8f6c6271/src/runtime_functions.js#L118

Any emscripten function to be called from C needs one. Probably this is an upstream bug.

Reply:
Hmm this doesn't seem to be necessary anymore?

Reply:
It does still seem necessary.

Reply:
Opened an issue here:
https://github.com/emscripten-core/emscripten/issues/17106

Reply:
Okay but "v" is also the wrong signature: the function takes zero arguments and returns a double. So if this function were ever actually called, we would get a crash.